### PR TITLE
Update logger to 2.1.0

### PR DESCRIPTION
**Pull request by mans0954**
### Description

Bump logger to 2.1.0

#### Related Issues / PRs

https://github.com/overleaf/google-ops/issues/1123

#### Manual Testing Performed

- [ ]
- [ ]

#### Deployment Checklist

- [ ] Update documentation not included in the PR (if any)
- [ ]

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3789,76 +3903,15 @@
...
-        "@overleaf/o-error": "^2.0.0",
+        "@overleaf/o-error": "^3.0.0",
         "bunyan": "1.8.12",
         "raven": "1.1.3",
-        "request": "2.88.0",
         "yn": "^3.1.1"
-      },
-      "dependencies": {
-        "bunyan": {
-          "version": "1.8.12",
```
Comment (das7pad) [inline package-lock.json:3805]: There is a `bunyan` package at the top level.

**Comment by mans0954** [comment]
Actually, I don't think this works as is - clsi is running on docker, not kubernetes, so won't pickup the configmap.

**Comment by mans0954** [comment]
I guess we either patch logger to fall back to using metadata if `/logging` isn't present on the file system (this isn't GKE so no problem with accessing metadata here) or patch the startup scripts to mount a volume and then somehow set a value in that from the rakefile. The latter sounds like more work, but might be more consistent. Thoughts? @das7pad @henryoswald @briangough

**Comment by mans0954** [comment]
> I guess we either patch logger to fall back to using metadata if `/logging` isn't present on the file system (this isn't GKE so no problem with accessing metadata here) or patch the startup scripts to mount a volume and then somehow set a value in that from the rakefile. The latter sounds like more work, but might be more consistent. Thoughts? @das7pad @henryoswald @briangough

For the latter option, I think we'd either have to scp the file to each host (and hope no new hosts are spun up during the logging period) or have shared storage mounted read-only by all the clsi, which seems a bit overkill.

**Comment by briangough** [comment]
> I guess we either patch logger to fall back to using metadata if `/logging` isn't present on the file system (this isn't GKE so no problem with accessing metadata here) or patch the startup scripts to mount a volume and then somehow set a value in that from the rakefile. The latter sounds like more work, but might be more consistent. Thoughts? @das7pad @henryoswald @briangough

I suggest falling back to the old way, but use an environment variable to explicitly select it  (so we don't start making unwanted metadata requests if the /logging directory is missing for unexpected reasons).

**Comment by jdleesmiller** [comment]
> providing an option to use the old way, with an environment variable to explicitly select it 

I think that is indeed what https://github.com/overleaf/logger-module/pull/23 settled on 👍

**Comment by mans0954** [comment]
Closed in favour of https://github.com/overleaf/clsi/pull/189

**Pull request closed without merging**